### DISCUSSION TITLE
JobPosting, RealEstateListing vs itemListElement

### DISCUSSION THREAD

**Discussion by ngterry** [schemaorg/schemaorg#3024] [Q&A]
Is there a more gereral term like Listing or Posting?  I have found some specific types like JobPosting and RealEstateListing, but I can't find a more general type for Posting or Listing.

What if there are "For Sale" or "Services, etc, is it recommended to use itemListElement and name attribute instead?  For example, in [Craigslist](https://newyork.craigslist.org/d/appliances/search/ppa?lang=en&cc=gb), the site is using itemListElement instead.

**Comment by philbarker** [comment]
@ngterry I think you are looking for [Offer](https://schema.org/Offer)
